### ISSUE TITLE
Turn off email notification

### ISSUE THREAD

**Issue by Veka85** [wekan/wekan#5407]
WeKan ® Version 	7.42.0
Meteor version 	2.14
Node version 	14.21.4
MongoDB version 	6.0.14
MongoDB storage engine 	wiredTiger
MongoDB Oplog enabled 	
OS Type 	Linux
OS Platform 	linux
OS Arch 	x64
OS Release 	3.10.0-1062.12.1.el7.x86_64

Is there a way to turn off email notifications in global, or even better for specific board?

**Comment by Veka85** [comment]
Tnx for the fast reply, bit if would be ideal if every user could have the option to manage this by himself (turn on/off email notifications)

**Comment by xet7** [comment]
@Veka85 

I think, WeKan already has that, at per-board Muted/Tracking/Watching.

**Comment by Veka85** [comment]
Yes, those are useful, i need those notifications in the notifications tab https://prnt.sc/WUmlV6pVKF6G , but would like an ability to deactivate incoming email notifications...
